CHANGELOG.md

```diff
@@ -3,20 +3,30 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v1.6.0] - 2023-06-21
```

Comment:
looks like changelog was stale. latest version is 1.9.0 - mind updating changelog and https://github.com/trycourier/courier-java/blob/main/pom.xml#L11 to 1.10.0?